apl-utils/src/main/java/com/apollocurrency/aplwallet/apl/util/Constants.java

```diff
@@ -191,6 +191,10 @@
     public static final int DEX_ETH_NUMBER_OF_CONFIRMATIONS = 10; // 150 sec for 15sec blocks
     public static final int DEX_APL_NUMBER_OF_CONFIRMATIONS = 30; // 150 sec for 5 sec blocks (average block time for 2/10 adaptive forging)
 
+    public static final int DEX_INTERVAL_MIN = 60; 
```

Comment:
Need more clarifications here, also better to include into constant name 'chart'|'graph'|'tv' to obviously relate it to the trading view feature.

Reply:
done